cmd/seccomp-operator/main.go

```diff
@@ -114,6 +115,10 @@ func run(*cli.Context) error {
 		return errors.Wrap(err, "create manager")
 	}
 
+	if err := seccompoperatorv1alpha1.AddToScheme(mgr.GetScheme()); err != nil {
+		return errors.Wrap(err, "Cannot add core Seccomp APIs to scheme")
```

Comment:
Let's start with lowercase errors and keep the description as short as possible to keep the noise low in the error stack:
```suggestion
		return errors.Wrap(err, "add core seccomp APIs to scheme")
```

Reply:
fixed